src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url= jdbc:postgresql://${DB_HOST}:${DB_PORT}/${DB_NAME}
+spring.datasource.username=${DB_USER}
+spring.datasource.password=${DB_PASS}
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.database-platform=postgres
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
```

Comment:
Note for later: remember to configure the db variables.
I think heroku has an option for hosting a postgres db (along with a web app). If not we would need to host the db somewhere.

Reply:
Heroku does fortunately - that's actually what we'll be doing in this week's lab

Reply:
heroku provides support for postgreSQL, see 
https://dev.to/prisma/how-to-setup-a-free-postgresql-database-on-heroku-1dc1

Reply:
Refer to #9